docs/source/models/other_models/pregnancy/gbd_2021_mncnh/index.rst

```diff
@@ -0,0 +1,335 @@
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - State
+    - Description
+    - Note
+  * - np
+    - Not pregnant or postpartum
+    - 
+  * - p
+    - Pregnant
+    - 
+  * - output
+    - Output data 
+    - Recording of simulant data for use in the next phase of the simulation, the intrpartum model.
+
+.. note::
+
+  For this model, we will not be using "time" or "time steps" in the traditional sense. Rather, simulants will be initialized and then go through the decision tree outlined on the :ref:`MNCNH Portfolio project <2024_concept_model_vivarium_mncnh_portfolio>` page. Following that, data will be recorded for use in the intrapartum model. 
+
+.. list-table:: State prevalence table for initialization
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - State
+    - Value
+    - Note
+  * - np
+    - 0
+    - 
+  * - p
+    - 1
+    - 
+
+.. list-table:: State transition data
+  :header-rows: 1
+
+  * - Source state
+    - Sink state  
+    - Transition name
+    - Transition rate
+    - Note
+  * - np
+    - p
+    - incidence_p
+    - 0
+    - Assumed zero for convenience. Note that this is typically :math:`\frac{ASFR + ASFR * SBR + incidence_\text{c995} + incidence_\text{c374}}{prevalence_\text{np}}`
+  * - p
+    - output
+    - completion of the decision tree
+    - Time step based transition where all simulants move to the next stage following the decision tree
+    - Single time step based state 
+
+.. list-table:: Data values
+  :header-rows: 1
+
+  * - Parameter
+    - Data type  
+    - Data ID
+    - Source
+    - Note
+  * - ASFR
+    - Covariate
+    - 13
+    - get_covariate_estimates: decomp_step='iterative' for GBD 2021
+    - Assume lognormal distribution of uncertainty.
+  * - SBR
+    - Covariate
+    - 2267
+    - get_covariate_estimates: decomp_step='iterative' for GBD 2021
+    - Parameter is not age specific and has no draw-level uncertainty. Use mean_value as location-specific point parameter.
+  * - incidence_c995
+    - Incidence rate of abortion and miscarriage cause
+    - c995
+    - como; decomp_step='iterative'
+    -  Use the :ref:`total population incidence rate <total population incidence rate>` directly from GBD and do not rescale this parameter to susceptible-population incidence rate using condition prevalence. 
+  * - incidence_c374
+    - Incidence rate of ectopic pregnancy
+    - c374
+    - como; decomp_step='iterative'
+    -  Use the :ref:`total population incidence rate <total population incidence rate>` directly from GBD and do not rescale this parameter to susceptible-population incidence rate using condition prevalence. 
+
+.. list-table:: Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
+     -
+   * - Female only
+     - True
+     -
+   * - Age group start
+     - 10 to 14 years
+     - ID=7
+   * - Age group end
+     - 50 to 54 years
+     - ID=15
+
+
+**Pregnancy modeling steps:**
```

Comment:
Why not make this an actual subsection so it shows up in the TOC?

Reply:
I think putting it at the ++++++++++++ level would be the appropriate thing (subsection of Vivarium modeling strategy, super-section of the two details subsections below, same level as Assumptions and limitations)